playbooks/de-populate-consul.yaml

```diff
@@ -254,6 +254,9 @@
         - {key: "configs/{{environment_name}}/de/redis-port", value: "{{ redis_port }}"}
         - {key: "configs/{{environment_name}}/de/redis-db-number", value: "{{ redis_db_number }}"}
         - {key: "configs/{{environment_name}}/de/redis-password", value: "{{ redis_password }}"}
+        - {key: "configs/{{environment_name}}/de/grouper-id", value: "{{ apps.grouper_user }}"}
```

Comment:
This looks like a duplicate of `configs/{{environment_name}}/apps/grouper-user`.

The DE configs should probably just read from the existing `apps/grouper-user` key, unless we want to allow the DE and apps service to use different grouper user IDs.